### PR TITLE
Added test for edit cart page

### PR DESCRIPTION
**Pull request by devpatil7**
Verification `vendor/bin/mftf run:test CartPageEditProductAndCheckout` should pass

### CODE REVIEW

File: app/code/Magento/Pwa/Test/Mftf/Test/VeniaCartPageEditCardAndCheckout.xml [hunk 1]
```diff
@@ -8,35 +8,34 @@
...
+            <title value="Edit product on cart page and checkout"/>
+            <description value="Edit product on cart page and checkout"/>
+            <severity value="CRITICAL"/>
+            <testCaseId value="MC-32298"/>
+            <group value="PWA"/>
+            <group value="Cart"/>
+        </annotations>
+
+        <before>
+            <actionGroup ref="GoToVeniaDesktopStorefront" stepKey="goToVeniaDesktopStorefront1"/>
```
Comment (revanth0212) [inline app/code/Magento/Pwa/Test/Mftf/Test/VeniaCartPageEditCardAndCheckout.xml:24]: Just curious, can we not have an action group to add an item to cart? I have seen this in multiple places.

File: app/code/Magento/Pwa/Test/Mftf/ActionGroup/VeniaCartPageActionGroup.xml [hunk 2]
```diff
@@ -147,4 +146,45 @@
...
+        <wait time="5" stepKey="temporaryWait"/>
+        <seeElement selector="{{EditPanelSection.updateCart}}" stepKey="checkForEditPanelDisplay"/>
+    </actionGroup>
+
+    <actionGroup name="CartPageEditProductConfigurations">
+        <arguments>
+            <argument name="product" defaultValue="ValeriaTwoLayerTank"/>
+        </arguments>
+        <waitForPwaElementVisible selector="{{EditPanelSection.colorByValue(product.color1)}}" stepKey="waitForProducttoDisplay"/>
+        <click selector="{{EditPanelSection.colorByValue(product.color1)}}" stepKey="clickOnColorOption01"/>
```
Comment (revanth0212) [inline app/code/Magento/Pwa/Test/Mftf/ActionGroup/VeniaCartPageActionGroup.xml:168]: Can this be a false positive? What if the color and size already selected are the first ones on the list?

**Review by revanth0212 — APPROVED** [review]

File: app/code/Magento/Pwa/Test/Mftf/ActionGroup/VeniaCartPageActionGroup.xml (hunk 2 above)
Comment (devpatil7) [reply app/code/Magento/Pwa/Test/Mftf/ActionGroup/VeniaCartPageActionGroup.xml:168]: No, its based on name of swatch.  data key="color1">Rain</data

File: app/code/Magento/Pwa/Test/Mftf/Test/VeniaCartPageEditCardAndCheckout.xml (hunk 1 above)
Comment (devpatil7) [reply app/code/Magento/Pwa/Test/Mftf/Test/VeniaCartPageEditCardAndCheckout.xml:24]: we do have one  actionGroup ref="AddToCart" stepKey="addToCart1"

**Pull request merged**